tdrs-backend/celery_start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# export REDIS_URI=rediss://xtJnXoZpeLNID24pQhpPb9HSr@master.prd-3fa9afec-b332-43c8-9a46-0aaa72c49d53.pw1qnn.usgw1.cache.amazonaws.com:6379
+echo starting celery
+celery -A tdpservice.settings worker -c 1 &
+sleep 5
+# TODO: Uncomment the following line to add flower service when memory limitation is resolved
+echo "REDIS_URI: $REDIS_URI"
+celery -A tdpservice.settings --broker=$REDIS_URI flower &
```

Comment:
where is the redis server? What do we set REDIS_URI to? local to celery?

Reply:
We should also open the port in Nginx to "flower", it will give the admin visibility and monitoring capabilities

Reply:
@jtimpe @raftmsohani what does opening NGINX port to flower accomplish? I didn't think the celery server had any routes to it or networking connections. Do we need to add these?

Reply:
flower is a monitoring tool - it may be helpful to open the ports in dev
https://flower.readthedocs.io/en/latest/

Reply:
okay, I have mapped flower to use port 8080 (since we don't need that port for naything else) and added the route for the celery app. https://tdp-celery-sandbox.app.cloud.gov/metrics works :)